bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -42,23 +42,75 @@ spec:
         - |
           #!/bin/bash
           set -euo pipefail
+          set -x
+          if [[ -f /usr/bin/id ]]; then
+            /usr/bin/id openvswitch
+          else
+            echo "id command not found"
+          fi
+          if [[ -d /run/openvswitch ]]; then
+            ls -al /run/openvswitch
```

Comment:
`ls -al /run/openvswitch/ || true`     trailing `/` forces treating it as a directory.

Reply:
I guess question here as well, when is `/var/run` not a symlink to `/run` ?